assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -184,11 +192,14 @@ function statusPoll(
   dispatch: Function,
   csrf: CsrfState,
   referrerAcquisitionData: ReferrerAcquisitionData,
+  paymentMethod: PaymentMethod,
+  participations: Participations,
 ) {
 
   if (pollCount >= MAX_POLLS) {
-    const url: string = addQueryParamToURL(routes.recurringContribPending, 'INTCMP', referrerAcquisitionData.campaignCode);
-    window.location.assign(url);
+    successfulConversion(participations);
+    dispatch(checkoutPending(paymentMethod));
+    return undefined;
```

Comment:
you could also write `return;`

Reply:
I can't because it returns  Promise<Response> lower down